plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/protobuf/TestKafkaProtobufWithSchemaRegistryMinimalFunctionality.java

```diff
@@ -140,6 +140,34 @@ public void testBasicTopicForInsert()
                 "Insert is not supported for schema registry based tables");
     }
 
+    @Test
+    public void testUnsupportedNestedDataTypes()
+            throws Exception
+    {
+        String topic = "topic-unsupported-nested";
+        assertNotExists(topic);
+
+        UnsupportedNestedTypes.schema message = UnsupportedNestedTypes.schema.newBuilder()
```

Comment:
Is it possible for us to create the `DynamicMessage` via descriptor and use them to populate it ?

Reply:
Protobuf static generator is used to workaround of the PR unrelated issue, when the parsing by `ProtobufUtils` of the `unsupported_nested.proto` throws:
```
com.google.protobuf.Descriptors$DescriptorValidationException: io.trino.protobuf.schema.nested_value_one: "NestedValue" is not an enum type.

	at com.google.protobuf.Descriptors$FieldDescriptor.crossLink(Descriptors.java:1587)
	at com.google.protobuf.Descriptors$FieldDescriptor.access$1400(Descriptors.java:1057)
	at com.google.protobuf.Descriptors$Descriptor.crossLink(Descriptors.java:1000)
	at com.google.protobuf.Descriptors$Descriptor.access$1200(Descriptors.java:648)
	at com.google.protobuf.Descriptors$FileDescriptor.crossLink(Descriptors.java:600)
	at com.google.protobuf.Descriptors$FileDescriptor.buildFrom(Descriptors.java:321)
	at com.google.protobuf.Descriptors$FileDescriptor.buildFrom(Descriptors.java:289)
	at io.trino.decoder.protobuf.ProtobufUtils.getFileDescriptor(ProtobufUtils.java:128)
	at io.trino.decoder.protobuf.ProtobufUtils.getFileDescriptor(ProtobufUtils.java:82)
	at io.trino.decoder.protobuf.ProtobufUtils.getFileDescriptor(ProtobufUtils.java:75)
....
```